tests/test_server.py

```diff
@@ -71,6 +79,44 @@ def mock_dockerhub(
             },
         }
 
+    @contextlib.contextmanager
+    def github_webhook_data(
```

Comment:
Rename to align with similar responses in `testcases.py`.

```suggestion
    def build_github_payload(
```